src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessToken/Services/IAccessTokenValidator.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+
+namespace Altinn.Common.AccessToken.Services
+{
+    /// <summary>
+    /// Interface to validate access token
+    /// </summary>
+    public interface IAccessTokenValidator
+    {
+        /// <summary>
+        /// Validates an access token
+        /// </summary>
+        /// <param name="token">The token to validate</param>
+        /// <returns>A boolean indicating the validity of the token</returns>
+        public Task<bool> Validate(string token);
```

Comment:
It is unnecessary with public here.

Reply:
Hmm, so it turns out that our Analyzer rules picking on this might be outdated. It became allowed in C# 8. Still unnecessary for public methods, but it is now possible to define methods as protected. And with alternative being explicit might have a purpose.